applications/serial_lte_modem/Kconfig

```diff
@@ -115,12 +115,6 @@ config SLM_DATAMODE_TERMINATOR
 	help
 	  Use a pattern to terminate data mode
 
-config SLM_DATAMODE_SILENCE
-	int "Silence time to exit data mode"
-	default 1
-	help
-	  Exit data mode after the specified time (in seconds) of UART silence
-
```

Comment:
could you please have such change in independent commit, as it is unrelated to the bug fix.

Also, there is a reference to this symbol here: https://github.com/nrfconnect/sdk-nrf/blob/main/applications/serial_lte_modem/doc/HTTPC_AT_commands.rst#set-command-1

which will now break when you remove this symbol from Kconfig.

Reply:
Yes, in this case, the kconfig in the docs will cease to link. Do we now need that sentence (which mentions the kconfig) in the docs?

Reply:
Thanks! Please allow this piggy-back change to catch up with v1.9.0.